build.py

```diff
@@ -1161,7 +1297,30 @@ def dockerfile_prepare_container_linux(argmap, backends, enable_gpu, target_mach
         fi \\
       && [ `id -u $TRITON_SERVER_USER` -eq 1000 ] \\
       && [ `id -g $TRITON_SERVER_USER` -eq 1000 ]
+""".format(
+        gpu_enabled=gpu_enabled
+    )
 
+    # This
+    if target_platform() == "rhel":
+        df += """
+# Common dpeendencies.
+RUN yum install -y \\
+        git \\
+        gperf \\
+        re2-devel \\
+        openssl-devel \\
+        libtool \\
+        libcurl-devel \\
+        libb64-devel \\
+        gperftools-devel \\
+        patchelf \\
+        wget \\
+        numactl-devel \\
+        wget
```

Comment:
```suggestion
        numactl-devel
```
Duplicate. Won't require new CI.